### PR TITLE
Transmission config flow

### PR DESCRIPTION
**Pull request by engrbm87**
**Description:**
Configure Transmission through config flow.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26434

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

File: source/_components/transmission.markdown [hunk 1]
```diff
@@ -31,6 +31,8 @@ If everything is set up correctly, the details will show up in the frontend.
 
 ## Configuration
 
+Home Assistant offers UniFi integration through **Configuration** -> **Integrations** -> **Transmission**. For legacy support old transmission configuration is imported and set up as new integration.
```
Comment (MartinHjelmare) [inline source/_components/transmission.markdown]:
```suggestion
Set up the integration through **Configuration** -> **Integrations** -> **Transmission**. For legacy support old transmission configuration is imported and set up as new integration.
```

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_components/transmission.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/transmission.markdown]: Applied suggestion.

**Comment by engrbm87** [comment]
I think we need to edit the description related to the `monitored_conditions` as they are no longer configured through configuration.yaml or imported.

**Comment by frenck** [comment]
@engrbm87 It can simply be removed, right?

**Comment by engrbm87** [comment]
> 
> 
> @engrbm87 It can simply be removed, right?

Yes we can remove it. We can mention that the following sensors will be automatically added. And we need to add the start/stop all torrents switch which wasn't there before.

**Comment by frenck** [comment]
Could you please update the documentation accordingly? Thanks :+1:

**Review by klaasnicolaas — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks for the adjustments @engrbm87! 👍

**Pull request merged**